apps/block_scout_web/assets/css/theme/_trustlines_variables.scss

```diff
@@ -72,3 +74,10 @@ $btn-qr-color: $primary; // btn qr-code
 // card
 $card-background-1: $primary;
 $card-tab-active: $primary;
+
+// Fix the dark mode
```

Comment:
What is fixed? how

Reply:
I added a fixup for that. Please let me know if this enough and understandable.
_Note:_ This comment was initially meant as a header for possible multiple fixes. Finally we only need this.

Reply:
We had a call and it was the outcome that we should try to solve the problems as the other networks do as well. I discovered that they have adopted their section `//links & tile`. Doing so removes the need of the section discussed here. It simply uses the "official way" instead of the custom styles and solves the problem as expected. Next time I should use this approach from the very beginning.